_docs_explore/22-security.md

```diff
@@ -9,6 +9,14 @@ News are full of hacking, spamming, skimming, phishing, or you-name-it attempts.
 
 Enmeshed itself handles very private and sensitive information. We are aware that security must be one of our topmost priorities. And by using Enmeshed, we think that the world would be a lot more secure.
 
+# Docker Compose File Security Considerations
+
+When using Docker Compose files, it is important to understand that the default configurations may not be secure for production use. As such, it is important to take appropriate security measures to ensure that your system is secure.
```

Comment:
@stnmtz ich glaube es wäre besser es nur dort zu schreiben wo wir wirklich compose files "anbieten". Hier wirkt es ein wenig aus dem Kontext gerissen.

Reply:
Ja und nein. Wir hatten gesagt, wir machen das zusätzlich in die Docker Compose files. Dass wir in den Security Recommendations noch mal schreiben, dass man bitte die Docker Compose files, die wir anbieten, nicht im Produktivbetrieb nutzt, ist denke ich trotzdem sinnvoll.

Reply:
Moment, jetzt verstehe ich. Ich würde das in https://enmeshed.eu/integrate/connector-security - also bei Integrate - reinhauen.

Reply:
Dann muss das aber anders formuliert werden, so finde ich es echt verwirrend :D

Reply:
Was haltet ihr von:
> Docker Compose is a tool to easily set up and host and complete landscape by running multiple Docker containers, configure them and link them together with a network. For development, testing and demonstration purposes, the enmeshed team provides Docker Compose files throughout this site or on GitHub. Please be aware, that those Docker Compose files should not be used in a public or productive environment, as they could contain insecure or otherwise unstable configurations, e.g. default passwords or the missing encryption at rest for MongoDB configuration.

Reply:
sehr gut!